images/kubekins-e2e/Dockerfile

```diff
@@ -33,9 +33,11 @@ RUN mkdir -p /go/src/k8s.io/kubernetes \
 # preinstall:
 # - graphviz package for graphing profiles
 # - bc for shell to junit
+# - mercurial for downloading third-party dependencies (git is already installed)
 # - rpm for building RPMs with Bazel
 RUN apt-get install -y bc \
     graphviz \
+    mercurial \
```

Comment:
should this be in kubekins-e2e or in bootstrap?

Reply:
that's more work, but you're right. :)

Reply:
time to bump all the things! we've needed a kubekins bump today anyhow for some firewall cleanup stuff